<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-4NIKV9T" name="Scenario_Component.Scenario_Phonenumber" tests="6" failures="1" timestamp="01 Aug 2018 13:34:05 GMT" time="103.149" errors="0">
  <testcase name="Permission" time="9.812" classname="Scenario_Component.Scenario_Phonenumber"/>
  <testcase name="Phoneno_UI" time="2.980" classname="Scenario_Component.Scenario_Phonenumber"/>
  <testcase name="Phoneno_Toast_Messages" time="59.922" classname="Scenario_Component.Scenario_Phonenumber">
    <failure type="java.lang.AssertionError" message="P:0/1 ~|dX:00 |dY:0.0 Xv:00 _Yv:00 ER RES
Fastrack REFLEX
PoweredBy (QV |
Register with your phone number

{
p
| agree to the Terms and Conditions
Please enter a valid phone number
SUBMIT
 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: P:0/1 ~|dX:00 |dY:0.0 Xv:00 _Yv:00 ER RES
Fastrack REFLEX
PoweredBy (QV |
Register with your phone number

{
p
| agree to the Terms and Conditions
Please enter a valid phone number
SUBMIT
 expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at PageObject_Component.PageObject_phonenumber.accept_tc_errortoast(PageObject_phonenumber.java:188)
at PageObject_Component.PageObject_phonenumber.verify_toastmessages(PageObject_phonenumber.java:175)
at Scenario_Component.Scenario_Phonenumber.Phoneno_Toast_Messages(Scenario_Phonenumber.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Phoneno_Toast_Messages -->
  <testcase name="Verify_Country_UI" time="4.279" classname="Scenario_Component.Scenario_Phonenumber"/>
  <testcase name="Verify_TermPDF" time="9.882" classname="Scenario_Component.Scenario_Phonenumber"/>
  <testcase name="Moveto_OTP" time="16.274" classname="Scenario_Component.Scenario_Phonenumber"/>
</testsuite> <!-- Scenario_Component.Scenario_Phonenumber -->
